app/views/layouts/admin.html.erb

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><%= Rails.application.class.module_parent_name %></title>
+    <title><%= content_for(:title) ? content_for(:title) : 'Admin' %> | <%= Setting.html_title_base %></title>
```

Comment:
is there a reason these two title conditions are expressed with two different code styles?

Reply:
Because the admin one is "${optional title || 'Admin'} | ${title base}", but the user one is "${optional title  | ${title base}", and the pipe separator is only present if there is something to separate

Reply:
The diff to make them more similar looks like:

```diff
diff --git a/app/views/layouts/admin.html.erb b/app/views/layouts/admin.html.erb
index 6fc8eb1..2a164de 100644
--- a/app/views/layouts/admin.html.erb
+++ b/app/views/layouts/admin.html.erb
@@ -3,7 +3,13 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><%= content_for(:title) ? content_for(:title) : 'Admin' %> | <%= Setting.html_title_base %></title>
+    <title>
+      <% if content_for(:title) %>
+        <%= content_for(:title) %>
+      <%- else %>
+        Admin
+      <%- end %> |
+      <%= Setting.html_title_base %></title>
     <%= csrf_meta_tags %>
     <%= action_cable_meta_tag %>
     <%= stylesheet_link_tag 'application', media: 'all' %>
```